src/hooks/useDrivesContainer.ts

```diff
@@ -31,30 +38,46 @@ export function getNodePath(node: Node, allNodes: Node[]): string {
     return path.join(getNodePath(parentNode, allNodes), encodeID(node.id));
 }
 
-function getBaseItemType(sharingType: string): BaseTreeItem['type'] {
+function getDriveBaseItemType(sharingType: string) {
     switch (sharingType.toLowerCase()) {
         case 'public':
-            return 'PUBLIC_DRIVE';
+            return PUBLIC_DRIVE;
         case 'shared':
-            return 'CLOUD_DRIVE';
+            return CLOUD_DRIVE;
         default:
-            return 'LOCAL_DRIVE';
+            return LOCAL_DRIVE;
     }
 }
 
-export function driveToBaseItems(
-    drive: DocumentDriveDocument,
-): Array<BaseTreeItem> {
+async function getSyncStatus(
+    driveId: string,
+    type: DriveType,
+): Promise<SyncStatus | undefined> {
+    if (getIsLocalDrive(type)) return;
+    try {
+        (await window.electronAPI?.documentDrive.getSyncStatus(
```

Comment:
This should be called using `useDocumentDriveServer`. `electronAPI` is not defined when Connect is run on the browser.
`useDocumentDriveServer` needs to provide a `getSyncStatus` method.

Reply:
ok cool will refactor